ContributedExamples/plot_mht.ipynb

```diff
@@ -4,27 +4,18 @@
    "cell_type": "markdown",
```

Comment:
<p>I don't think the configurations argument to <code>get_experiments</code> here actually does anything? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I think it is just showing how to check for experiments available in ACCESS. I can remove it if it is not needed.